src/views/Dashboard.vue

```diff
@@ -1,42 +1,38 @@
 <template>
-  <el-row>
-    <el-col :span="6">left</el-col>
+  <el-col :span="6" class="padding">
+    <OwnRepositories v-if="githubToken && user"/>
+    <div v-if="!githubToken || !user">Not logged in</div>
+  </el-col>
 
-    <el-col :span="12">
-      <ul class="infinite-list" v-infinite-scroll="load" v-if="adoptables">
-        <li v-for="(adoptable, index) in adoptables" :key="`adoptable:${index}-${adoptable.repository}`">
-          <Adoptable :repository="adoptable.repository" :description="adoptable.description" :readme="adoptable.readme" />
-        </li>
-      </ul>
-    </el-col>
+  <el-col :span="12">
+<!--TODO make it so correct items is loaded depending on what the user want to see-->
```

Comment:
Isn't this todo related to the infinite list itself -> move comment

Reply:
No, this is depending on the page, i'll change the comment to make it more clear